pkg/command/service/measure.go

```diff
@@ -681,6 +682,7 @@ kperf service measure --svc-perfix svc --range 1,200 --namespace ns --concurrenc
 	serviceMeasureCommand.Flags().StringVarP(&measureArgs.namespaceRange, "namespace-range", "", "", "Service namespace range")
 	serviceMeasureCommand.Flags().StringVarP(&measureArgs.namespacePrefix, "namespace-prefix", "", "", "Service namespace prefix")
 	serviceMeasureCommand.Flags().IntVarP(&measureArgs.concurrency, "concurrency", "c", 10, "Number of workers to do measurement job")
+	serviceMeasureCommand.Flags().StringVarP(&measureArgs.output, "output", "o", "/tmp", "Measure result location")
```

Comment:
Why not default the output to current location, i.e., “.” Since `/tmp` may not exist or be writable

Reply:
Because "/tmp" is current default output dir, so I thought we should not change it. Should I change the current default dir to .?

Reply:
+1, I think setting `corrent location` as default make more sense.

Reply:
I have changed to the current location as default.